README.md

```diff
@@ -6,6 +6,12 @@
 
 DDL module for Tarantool 1.10+
 
+The DDL module allows you to describe a data schema in the YAML format in a
+declarative way. It's alternative way to describe data schema using Lua,
+it is simpler to use, and you do not have to dive deep into Lua.
```

Comment:
```suggestion
The DDL module allows you to describe a data schema in the YAML format in a
declarative way. It's an alternative way to specify the schema rather than in
Lua. It's simpler to use, and you do not have to dive deep into Lua.
```

Reply:
The DDL module enables you to describe data schema in a declarative YAML-based format.
It is a simpler alternative to describing data schema in Lua and doesn't require having a deep knowledge of Lua.